Gopkg.toml

```diff
@@ -15,6 +15,7 @@ required = [
   "github.com/openshift/api/config/v1",
   "github.com/onsi/ginkgo",
   "github.com/onsi/gomega",
+  "sigs.k8s.io/controller-tools/cmd/controller-gen", # for crd/rbac generation
```

Comment:
couldn't we `go get` here or similar so no need to bring +29,122 lines?

Reply:
That's the way it is. `go get` would not help if we want to keep all the deps in `Gopkg.toml`.

Reply:
which deps? this is not used in code right? We need a `controller-gen` binary to leverage automation for the case where we want to regenerate CRDs. Why can't we fetch it instead of building it from source code?

Reply:
I think we should keep it in `vendor` and add a step to the build that re-generates the CRDs and fails if a diff is found.

Reply:
I am gonna ignore this discussion then